src/common/models/Projection.ts

```diff
@@ -283,6 +319,8 @@ export class Projection {
     return this.metrics?.testPositivityRatioDetails?.source || null;
   }
 
+  // TODO: Add new metrics to Metric Enum, add to getMetricValue() and resolve any issues
+  // from adding new metrics to Metric.
```

Comment:
As mentioned in Slack, if you merge the latest `can-82` code I think we'll have the `Metric` enum values we need.